codenames-front-end/src/components/game/stopped/menu/CNStopGameMenu/CNStopGameMenu.tsx

```diff
@@ -0,0 +1,81 @@
+import React, {FC, useEffect} from 'react';
+import {IGameRoom} from "../../../../../models/CodeNames/IGameRoom";
+import css from "./CnStopGameMenu.module.css";
+import {Color} from "../../../../../models/CodeNames/Color";
+import CodeNamesWebSocketService from "../../../../../services/CodeNamesWebSocketService";
+import {Status} from "../../../../../models/CodeNames/Status";
+import CNSpectateBlock from "../settings/CNSpectateBlock/CNSpectateBlock";
+import CNTeam from "../CNTeam/CNTeam";
+import AdminControl from "../settings/AdminControl/AdminControl";
+
+interface CNStopGameMenuProps {
+    room?: IGameRoom;
+    service?: CodeNamesWebSocketService;
```

Comment:
Probably it will be better to use custom hooks to handle websocket connection
Using services to contain logic is not really React-ish

Reply:
It's more convenient, I changed it to a hook